Lib/Src/CAdcData.cpp

```diff
@@ -48,7 +48,7 @@ void CAdcData::init()
  * @param adc_channel Number of channel starting from 0
  * @retval Volts
  */
-float CAdcData::operator[](uint8_t adc_channel)
+float CAdcData::operator[](uint8_t adc_channel) const
 {
     // Check channel is within defined range
```

Comment:
For line 60 which is not part of this MR however useful to add: 
C++ style casts are checked by the compiler. C style casts aren't and can fail at runtime.